pulumi/Pulumi.dev.yaml

```diff
@@ -0,0 +1,6 @@
+config:
+  aws:region: us-east-2
+  crc-pulumi:instance_ami: ami-0521a1ab6cb98215d
+  crc-pulumi:instance_type: t2.micro
+  crc-pulumi:key_name: pulumi-crc-key-pair
+  crc-pulumi:region: us-east-2
```

Comment:
Here too. Is there a need for duplicate region keys?

Reply:
Indeed not needed as of now